spec/lob/resources/postcard_spec.rb

```diff
@@ -37,12 +38,38 @@
         description: @sample_postcard_params[:description],
         to: new_address["id"],
         message: @sample_postcard_params[:message],
-        front: "https://lob.com/postcardfront.pdf",
+        front: "https://lob.com/postcardfront.pdf"
       )
 
       result["description"].must_equal(@sample_postcard_params[:description])
     end
 
+    it "should accept an idempotency_key" do
+      new_address = subject.addresses.create @sample_address_params
+
+      uuid = SecureRandom.uuid
+
+      new_postcard = subject.postcards.create({
+        description: @sample_postcard_params[:description],
+        to: new_address["id"],
+        message: @sample_postcard_params[:message],
+        front: "https://lob.com/postcardfront.pdf"
+      }, {
+        "Idempotency-Key" => uuid
```

Comment:
Does this `Idempotency-Key` header need to have a particular format?

If it can be any string, I suggest that we just remove the `SecureRandom` gem and make it something like `Test-Idempotency-Header`.

If we do need it to be a random UUID, then we will need to add `SecureRandom` as a dev dependency to the `lob.gemspec` file with the following line:

```ruby
spec.add_development_dependency "bundler", "~> 2.0.0"
```
I would prefer the first if we can (since it removes the need for another dependency), but I'm not sure if we require a specific format.